src/objects/core/admin.py

```diff
@@ -13,6 +18,32 @@ class ObjectTypeAdmin(admin.ModelAdmin):
     )
     readonly_fields = ("_name",)
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "<int:objecttype_id>/versions/",
+                self.admin_site.admin_view(self.versions_view),
+            )
+        ]
+        return my_urls + urls
+
+    def versions_view(self, request, objecttype_id):
```

Comment:
Could you add two tests to verify that non authenticated and non staff users cannot access this endpoint?